src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const docPopulationArray = document.querySelectorAll('.population');
+
+const docTotal = document.querySelector('.total-population');
+
+const docAverage = document.querySelector('.average-population');
+
+const populationArray = [];
+
+docPopulationArray.forEach((item) => populationArray.push(item.textContent));
```

Comment:
You are creating an intermediate array 'populationArray' just to store text content, which is then transformed into numbers in 'populationNumbers'. You can streamline this by mapping directly from 'docPopulationArray' to 'populationNumbers', eliminating the need for 'populationArray'.